CRITICAL - 2023-09-24 18:14:48 --> syntax error, unexpected token ";", expecting ")"
in APPPATH\Controllers\Login_controller.php on line 31.
 1 SYSTEMPATH\Autoloader\Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\tpintegrador\\app\\Controllers\\Login_controller.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Login_controller')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Login_controller')
 4 SYSTEMPATH\CodeIgniter.php(890): class_exists('\\App\\Controllers\\Login_controller', true)
 5 SYSTEMPATH\CodeIgniter.php(484): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-24 18:15:26 --> syntax error, unexpected token ";", expecting "]"
in APPPATH\Controllers\Login_controller.php on line 43.
 1 SYSTEMPATH\Autoloader\Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\tpintegrador\\app\\Controllers\\Login_controller.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Login_controller')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Login_controller')
 4 SYSTEMPATH\CodeIgniter.php(890): class_exists('\\App\\Controllers\\Login_controller', true)
 5 SYSTEMPATH\CodeIgniter.php(484): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-24 18:17:14 --> syntax error, unexpected token "}", expecting ";"
in APPPATH\Controllers\Login_controller.php on line 73.
 1 SYSTEMPATH\Autoloader\Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\tpintegrador\\app\\Controllers\\Login_controller.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Login_controller')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Login_controller')
 4 SYSTEMPATH\CodeIgniter.php(890): class_exists('\\App\\Controllers\\Login_controller', true)
 5 SYSTEMPATH\CodeIgniter.php(484): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-24 18:18:07 --> Undefined variable $db
in APPPATH\Controllers\Login_controller.php on line 20.
 1 APPPATH\Controllers\Login_controller.php(20): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'C:\\xampp\\htdocs\\tpintegrador\\app\\Controllers\\Login_controller.php', 20)
 2 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Login_controller->auth()
 3 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login_controller))
 4 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-24 18:18:38 --> Undefined variable $password
in APPPATH\Controllers\Login_controller.php on line 37.
 1 APPPATH\Controllers\Login_controller.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $password', 'C:\\xampp\\htdocs\\tpintegrador\\app\\Controllers\\Login_controller.php', 37)
 2 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Login_controller->auth()
 3 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login_controller))
 4 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-24 18:19:42 --> Undefined variable $password
in APPPATH\Controllers\Login_controller.php on line 37.
 1 APPPATH\Controllers\Login_controller.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $password', 'C:\\xampp\\htdocs\\tpintegrador\\app\\Controllers\\Login_controller.php', 37)
 2 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Login_controller->auth()
 3 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login_controller))
 4 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-24 18:24:22 --> Class "App\Filters\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(471): CodeIgniter\Filters\Filters->run('panel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-24 18:29:03 --> Class "App\Filters\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(471): CodeIgniter\Filters\Filters->run('panel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-24 18:29:06 --> Class "App\Filters\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(471): CodeIgniter\Filters\Filters->run('panel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-24 18:29:08 --> Class "App\Filters\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(471): CodeIgniter\Filters\Filters->run('panel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-24 18:33:01 --> Class "App\Filters\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(471): CodeIgniter\Filters\Filters->run('panel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-24 18:33:29 --> Class "App\Filters\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(471): CodeIgniter\Filters\Filters->run('panel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-24 18:39:24 --> Class "App\Filters\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(471): CodeIgniter\Filters\Filters->run('panel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-24 18:39:26 --> Class "App\Filters\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(471): CodeIgniter\Filters\Filters->run('panel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-24 20:14:10 --> syntax error, unexpected token "}", expecting ";"
in APPPATH\Controllers\Login_controller.php on line 34.
 1 SYSTEMPATH\Autoloader\Autoloader.php(291): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\tpintegrador\\app\\Controllers\\Login_controller.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(269): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Login_controller')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Login_controller')
 4 SYSTEMPATH\CodeIgniter.php(890): class_exists('\\App\\Controllers\\Login_controller', true)
 5 SYSTEMPATH\CodeIgniter.php(484): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-09-24 22:20:41 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'valdivia_j' in C:\xampp\htdocs\tpintegrador\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\tpintegrador\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'valdivia_j', 3306, '', 0)
#1 C:\xampp\htdocs\tpintegrador\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tpintegrador\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tpintegrador\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tpintegrador\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tpintegrador\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('gime@gmail.com', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\tpintegrador\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('email', 'email', 'gime@gmail.com', Array, Array)
#7 C:\xampp\htdocs\tpintegrador\system\Controller.php(141): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\tpintegrador\app\Controllers\usuario_controller.php(20): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\tpintegrador\system\CodeIgniter.php(932): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\tpintegrador\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\tpintegrador\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\tpintegrador\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'valdivia_j' in C:\xampp\htdocs\tpintegrador\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\tpintegrador\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tpintegrador\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tpintegrador\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tpintegrador\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tpintegrador\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('gime@gmail.com', 'email', Array, NULL, 'email')
#5 C:\xampp\htdocs\tpintegrador\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('email', 'email', 'gime@gmail.com', Array, Array)
#6 C:\xampp\htdocs\tpintegrador\system\Controller.php(141): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\tpintegrador\app\Controllers\usuario_controller.php(20): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\tpintegrador\system\CodeIgniter.php(932): App\Controllers\usuario_controller->formValidation()
#9 C:\xampp\htdocs\tpintegrador\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#10 C:\xampp\htdocs\tpintegrador\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tpintegrador\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-09-24 22:20:41 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'valdivia_j'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('gime@gmail.com', 'email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('email', 'email', 'gime@gmail.com', [...], [...])
 6 SYSTEMPATH\Controller.php(141): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\usuario_controller.php(20): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\usuario_controller->formValidation()
 9 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
10 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-09-24 22:21:01 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'valdivia_j' in C:\xampp\htdocs\tpintegrador\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\tpintegrador\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'valdivia_j', 3306, '', 0)
#1 C:\xampp\htdocs\tpintegrador\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tpintegrador\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tpintegrador\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tpintegrador\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tpintegrador\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\tpintegrador\app\Controllers\Login_controller.php(25): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\tpintegrador\system\CodeIgniter.php(932): App\Controllers\Login_controller->auth()
#8 C:\xampp\htdocs\tpintegrador\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login_controller))
#9 C:\xampp\htdocs\tpintegrador\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tpintegrador\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'valdivia_j' in C:\xampp\htdocs\tpintegrador\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\tpintegrador\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tpintegrador\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tpintegrador\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tpintegrador\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tpintegrador\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\tpintegrador\app\Controllers\Login_controller.php(25): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\tpintegrador\system\CodeIgniter.php(932): App\Controllers\Login_controller->auth()
#7 C:\xampp\htdocs\tpintegrador\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login_controller))
#8 C:\xampp\htdocs\tpintegrador\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tpintegrador\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-24 22:21:01 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'valdivia_j'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login_controller.php(25): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Login_controller->auth()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login_controller))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
